### PR TITLE
  Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by naolkidanu**
Milestone:
- Add work section
- Add flex to cards
- Add CSS grid to work section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -80,6 +83,72 @@ <h1 class="main-description">
           </li>
         </ul>
     </section>
+    <section class="project">
+    <div class="card">
+      <img class="pic" src="img/Snapshoot Portfolio.png" alt="First-img" />
+      <p class="card-name">Tonic</p>
```
Comment (aimalamiri) [inline index.html]: Good job for implementing the same font, but it's highly recommended to use header tags for the titles, so please make sure to put the project name into a `h[1-6]` tag.

File: index.html [hunk 2]
```diff
@@ -80,6 +83,72 @@ <h1 class="main-description">
...
+        <li class="bak">Back End Dev</li>
+        <li class="bak">2015</li>
+      </ul>
+      <p class="tex-sec">A daily selection of privately personalized read;no account or signs-up required</p>
+      <ul class="work-list">
+      <li class="work-name">html</li>
+        <li class="work-name">css</li>
+        <li class="work-name">javascript</li>
+      </ul>
+      <button class="sec-button">see project</button>
```
Comment (aimalamiri) [inline index.html:99]: The design of your button looks great, and you can add some effects to improve it. For example, the hover, active, and disabled states, as you can see in the below screenshot from Figma.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @naolkidanu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The header section is designed very well and look fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️

- [ ] **Your project looks amazing, but according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) you must also add the `About myself` section. So make sure to have it in your next commit.**


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Tushar151997 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Good Redme✅ 
- Great Design✅ 
- Linters are passing ✅


 ## Required Changes ♻️

- [ ] **_🛑 as mentioned by the previous reviewer & according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) you must also add the `About myself` section and not only the project  section🛑_** 

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @naolkidanu,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested !! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by naolkidanu** [comment]
Okay, Thank you for your supportive response.

**Pull request merged**